packages/core/scripts/gas-rebate/rebates/Rebate_1.json

```diff
@@ -1,50 +1,152 @@
-    "0x2ea5b7c369cb88c171466bb6dcf3cbe1c508fbea": 0.4558121221578243,
-    "0x49a8a045eae2c9d4a7c7e2919ba957064bd59252": 0.4557702318323673,
-    "0x996d5138843993cebc554c20d82310a43802aff1": 0.4996904926437806,
-    "0xfec2ade141272f5271febadfca5d21ca30fc04a5": 0.4996904926437806,
-    "0x49739691fb5f3992b3f2536f309d955558e75933": 0.4207150111457869,
-    "0xc1228273c77a4939bcc42a0d8f7b0077228e7f3c": 0.4996695474810522,
-    "0xed6bf5eeffdcff967184747648f1c3b578a2f430": 0.489416890325457,
-    "0x1b52a70616ee7ed4ea070d19fd18a60e467a5276": 0.330259835042354,
-    "0xe7a6d81a4b543c1bf2bd10ceb74a72689dc2e4b1": 0.40359234061524746,
-    "0xd2a78bb82389d30075144d17e782964918999f7f": 0.4996695474810522,
-    "0x9ae57ce31bdae73a0d3bf701998af5494a1ceeec": 0.489416890325457,
-    "0x230d31eec85f4063a405b0f95bde509c0d0a8b5d": 0.41044140882746327,
-    "0xdb2808ed6cf0b8b9686222689cf4f6e88e42b80a": 0.5099640949621043,
-    "0xbe3bf9bb42bf423695b792250be206c9a58e2de1": 0.49964860231832364,
-    "0xf288464959a5dd1671bce3458a3a4ed0cc314e3c": 0.489416890325457,
-    "0x6888ff4be3262f3232e1c022de731680a16ddda7": 0.49964860231832364,
-    "0x605acc13c07cb2de5261dc64d315857fde7d5c5c": 0.4001678065091395,
-    "0xf627e5f4bad95a956468d8bb6ee20b119f992e96": 0.41386594293357115,
-    "0xde3af8bb46d556df3d7c99cc313c4b5713b4a5a5": 0.5245140013374944,
-    "0xbf87c98f8a9c1f2a88b613813ec1e974105eb615": 0.5450612059741418,
-    "0x14a29ccfe3c1cc777a6f3a2ead6f07cec5cc5dc1": 0.5450612059741418,
-    "0xf7b10d603907658f690da534e9b7dbc4dab3e2d6": 0.7635454346856888,
-    "0x1757d400578cf8a2f3da8228e4402fb1bb3fc968": 0.24284170753455195,
-    "0x67d24f48f1b58c4d4dcda57ba2386dce76353fa2": 0.2847878867588052,
-    "0x8d07d225a769b7af3a923481e1fdf49180e6a265": 0.24969077574676773,
-    "0x4c39185a078b5666c372538231cb793a0928807b": 0.25227750334373605,
-    "0xa66b7e13467084f248e7d731eb2f9ff8e43db1cf": 0.24284170753455195,
-    "0x56d14fcb4587b5845fa4823af790672ebf92580d": 0.24284170753455195,
-    "0x966cf5cd0624f1efcf21b0abc231a5ccc802b861": 0.25227750334373605,
-    "0x120219511b9b0254e7d15014db83b8752289fda3": 0.2847878867588052,
-    "0x57c458c1354fee3aba912794709c91180869f953": 0.24969077574676773,
-    "0xedc63f732d61df2556fd3ae84f911a84ac64b4ba": 0.24279981720909496,
-    "0xb2a5e135892b92d8554dab9adc5291ee65a9957b": 0.24284170753455195,
-    "0xe311958ac79b41bf3a0595470a964b8efbe0fb36": 0.29163695497102093,
-    "0x28d804bf2212e220bc2b7b6252993db8286df07f": 0.25912657155595187,
-    "0xbee33ac5520a2245cbf8288e768a5cc26927cddf": 0.24969077574676773,
-    "0xccd9c5465fac5c0df86fd9b7d524a5949defcef6": 0.24969077574676773,
-    "0xe6662ec7fc7c84231b29f0f0da6321673d100854": 0.24284170753455195,
-    "0x8bd16de8938a2ad16794ac7e7502896f156e370c": 0.24969077574676773,
-    "0x2f840532892f91811f99cdaec703f322a779cc66": 0.22693734730271958,
-    "0xef22364cdabd7f39a3b6fd836f059336c810a8a9": 0.24969077574676773
+    "0x797e1f1765a0071454f443114bf26a53f798c071": 14.165486990510685,
```

Comment:
wow some of these numbers are really high. These units are in UMA, right? Some wallets getting as much as 14 UMA for gas rebates on 3 votes? That's like 224 USD at current prices.

Reply:
The higher gas usages are for the first reveal and the first claim. The first reveal also calls `snapshot()` and the first claim calls `resolvePrice()`. But yep, these are all based on the specific reveal and claim transactions here:

```
{
    "0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598832223,
        "revealGasUsed": 218727,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xe955caad0a21af7fda41fa8fccdd668cc4f4b541d677e4ce8244474db7423cb6",
        "commitTimestamp": 1598765441,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x2317faefba49d6cfd2f78c4fe052de4e5fdc216eecfc13c3aad005c389c84f46",
        "ethToPay": 0.0656287267810238,
        "umaToPay": 3.5559811195086555
    },
    "0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598832603,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xcaaef7821a42cde839ebd050a7ed9812180c3b23b3b347d166f6dc38b1c266c1",
        "commitTimestamp": 1598753809,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x767b34c92412e6448b8dca432d483f5a5f3b18b37aff26185d5ae7aa55a11dd7",
        "ethToPay": 0.0308712213620286,
        "umaToPay": 1.6727047085010271
    },
    "0x1757D400578cF8A2F3Da8228e4402FB1bb3fc968-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598833456,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x18461d5a5c6c5c72381742ec7baf776ca0036fa7cd468f9bc70248da0393e2d5",
        "commitTimestamp": 1598830232,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xcda6acaa8478f86470293c740f15c01de25e4b5ebedf4b0cb5e5f025e642817f",
        "ethToPay": 0.030407945372680305,
        "umaToPay": 1.6476028856858305
    },
    "0x49739691fB5f3992b3f2536f309D955558e75933-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598833851,
        "revealGasUsed": 73489,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x20bf71f74ab72744d1a04370e1cb9be7148e9167874ab727073d3fe922566743",
        "commitTimestamp": 1598787868,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x1a9dfde0207943bc777a1987e6b02c02a84a33b672a1f0d864dddb55ec8fae8c",
        "ethToPay": 0.03133449735137689,
        "umaToPay": 1.697806531316224
    },
    "0x9ae57ce31BDae73A0d3BF701998Af5494a1cEeec-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598835217,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xde1f3f0b6cb6a724895e195dc75c62a7899689a705e1bf3b2ba16ea81a05024d",
        "commitTimestamp": 1598810904,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x0608dea3c7d36d7451a5265a1eee73dac2cad5fb8379198f5388099537a28a73",
        "ethToPay": 0.030410488162649792,
        "umaToPay": 1.6477406624425326
    },
    "0x797E1f1765a0071454f443114Bf26a53F798c071-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598835229,
        "revealGasUsed": 81581,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xf49367482b59f166ecb5b67b280922a0c19396deaa0905aaf30648bb39bf185a",
        "commitTimestamp": 1598761329,
        "commitGasUsed": 76456,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x0c3ed5cf150b65e236c64885fccba204581f30ee4036aec46f6b29393fee5b8d",
        "ethToPay": 0.03546422369869829,
        "umaToPay": 1.9215687409475481
    },
    "0x67D24f48f1B58C4d4DcDA57Ba2386DcE76353fA2-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598835626,
        "revealGasUsed": 81581,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xad6074285cc8b8c5cafe2fdf2962964efb4b078f8dd2fe36dd3bdb08f78c68a1",
        "commitTimestamp": 1598811438,
        "commitGasUsed": 76456,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x4f6a8074b074982ca4dadce8efb7a336b50c086e1e7efb60feabd19a3dd751df",
        "ethToPay": 0.03546422369869829,
        "umaToPay": 1.9215687409475481
    },
    "0x605ACc13c07CB2dE5261dc64D315857fDE7d5C5c-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598837381,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xad07a30b4928bb4603ee5199cae9bf51bff373a0c5c60fac6d3d060d33df3f44",
        "commitTimestamp": 1598825225,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xf924e94f101bcf0aaa06c62e39b5d1f209972d9234768c80f2915869708c7f97",
        "ethToPay": 0.030407945372680305,
        "umaToPay": 1.6476028856858305
    },
    "0x8d07D225a769b7Af3A923481E1FdF49180e6A265-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598838304,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x4edca07f35a5ec76b64c8cf209ae48c3c490f737ac5a5a4be4e4f25f5f675a25",
        "commitTimestamp": 1598758768,
        "commitGasUsed": 65960,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xc80e6e4fb4fdad3dc55ea826c9757ee9d6de36fd2babd8c78b37b40bc35a8b97",
        "ethToPay": 0.03086613578208962,
        "umaToPay": 1.6724291549876231
    },
    "0xBf87C98F8A9c1F2A88b613813Ec1E974105EB615-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598838365,
        "revealGasUsed": 83543,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x1b5b08255ca0720d061b203608f178b6018c5b9a16dab6a73a8540f238bb1a2e",
        "commitTimestamp": 1598813314,
        "commitGasUsed": 76420,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xb212f2aab2151175a9ce332438bfdb2b28bda54737b87c5e3180efeb6a16da1b",
        "ethToPay": 0.03591987131813812,
        "umaToPay": 1.9462572334926387
    },
    "0x4c39185a078B5666C372538231cB793A0928807b-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598839365,
        "revealGasUsed": 72268,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xc6f8702ec87b3cda6a0ddd7497e34d3386ded8bfc2f4714c6efedcf459bb363c",
        "commitTimestamp": 1598758472,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xeb07c022915e7a9ef11d5ef04d2526ef7e6bdf4e07fd816f62dd6fd1f8bb4cde",
        "ethToPay": 0.031048732300880393,
        "umaToPay": 1.6823228372995545
    },
    "0xa66B7E13467084f248E7d731eb2f9FF8E43dB1CF-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598839897,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xb639f2e83d717efb76e61706a7a6128299f7c27265d39ae66d3069cb80f864a8",
        "commitTimestamp": 1598831114,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x8d77729c118a3130cd1aa504b3dee7046866cd59d6e94a9aa2ad070a8c787d13",
        "ethToPay": 0.030410488162649792,
        "umaToPay": 1.6477406624425326
    },
    "0x6888FF4bE3262F3232e1c022De731680a16dDdA7-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598840855,
        "revealGasUsed": 71515,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x5533cbafd96ab140648c89767c2ee6a61fad2e9bee2c5dc28a2122b9624cf21f",
        "commitTimestamp": 1598826839,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x3e1afc78e6eb6c7c597be9ac7c18bf06e3082a74a7f746dce2a608b5b1f15645",
        "ethToPay": 0.030868387869739007,
        "umaToPay": 1.6725511805327387
    },
    "0x56D14fcb4587B5845fA4823aF790672EBF92580d-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598841893,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x70132d5e1f24fc5e3ed9c2308c921ff8d2143781994946c827b05da9f1444e3d",
        "commitTimestamp": 1598797951,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xb0eee9ccf0a94877de9df1844d4f4a454ce200e89140c570ed779254da6c8126",
        "ethToPay": 0.030407945372680305,
        "umaToPay": 1.6476028856858305
    },
    "0x966Cf5cd0624f1EfCf21B0abc231A5CcC802B861-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598842375,
        "revealGasUsed": 72268,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x2972c871b5eebaaa4ad93421dd6a30368edb5d8c68931765155f1b670d98bc85",
        "commitTimestamp": 1598831551,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x87ecd0fe8dc1a1ae6483e37f523c9860736b2451f3fe1418dd313930f25f6abd",
        "ethToPay": 0.031046189510910906,
        "umaToPay": 1.6821850605428523
    },
    "0xD2A78Bb82389D30075144d17E782964918999F7f-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598843609,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xa1867a4bb4e3022a539144b6b37f134530a12395aac6594b30fdf8d5bd0b0090",
        "commitTimestamp": 1598764918,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xa19551ad48ee873b6d29c41bdf3708de961192a332a7013bbf838175a542aa25",
        "ethToPay": 0.030873764151998086,
        "umaToPay": 1.6728424852577293
    },
    "0x14A29ccfE3C1cc777A6f3a2ead6F07ceC5cC5dc1-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598846025,
        "revealGasUsed": 83543,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xbabcb333506bfa8d02a029c4553e0e9f830bced88e1055555de0104926c2f814",
        "commitTimestamp": 1598813362,
        "commitGasUsed": 76444,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xdd60934d52ae403f1e6177c2901eaba85bd6dc82edcce8b78cb30574b423e290",
        "ethToPay": 0.03592495689807709,
        "umaToPay": 1.946532787006043
    },
    "0xfEc2Ade141272f5271FEbaDFca5d21Ca30fc04a5-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598846748,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xe859f464c5cb07d2bbc3ee84b9c239e4b6185760c8900cfd92caba1b226e91b5",
        "commitTimestamp": 1598799042,
        "commitGasUsed": 65960,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x36fab2c51b4ffcd451ce338a17a65947f54e274044eff8465ed71fedd70abeb5",
        "ethToPay": 0.03086613578208962,
        "umaToPay": 1.6724291549876231
    },
    "0x120219511b9B0254E7D15014dB83B8752289FdA3-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598849308,
        "revealGasUsed": 81581,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xf40b506ec060e2d2bc271483b432df469748871881f24305725491130d541bd9",
        "commitTimestamp": 1598753632,
        "commitGasUsed": 76444,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xe0b95d302f9f232c691eea9191fb4fc786b8a03fb9b5f21ce39c4f022770cb3f",
        "ethToPay": 0.0354616809087288,
        "umaToPay": 1.921430964190846
    },
    "0xed6bf5EEfFdcFf967184747648F1C3b578a2f430-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598849453,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x54b44e1527159e11314146cd65ad6a9a25a38424b7bfb366469e2a3a83fab85f",
        "commitTimestamp": 1598773310,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x43e2b3f9ebcaad0dfaced48daa01799a474366ad27e660a2a3854c900bdef879",
        "ethToPay": 0.030410488162649792,
        "umaToPay": 1.6477406624425326
    },
    "0xc1228273C77a4939bCc42A0D8f7B0077228e7F3c-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598852184,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x2c6f3f889b3fc7e56f1170985ee22cb28ce61f6fcda41d57ac86edeea1401d84",
        "commitTimestamp": 1598773098,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x8d66b7a4919988cea5ef2707a6ecf528b815e8336d8d3b9d09754bbf36f85013",
        "ethToPay": 0.0308712213620286,
        "umaToPay": 1.6727047085010271
    },
    "0x57C458c1354fEE3ABA912794709C91180869f953-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598852406,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x1ecd5bb8449d1cfa3bd484b645eb0f32e4d2a987328f32037a2d42bef4df23ae",
        "commitTimestamp": 1598773613,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x7e3cc4ea0f0dbe96d3bab003a35e533553fcd45e5514eb1b5a4d3a50b2388715",
        "ethToPay": 0.030873764151998086,
        "umaToPay": 1.6728424852577293
    },
    "0x996d5138843993Cebc554c20d82310a43802aFF1-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598853069,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x329e6f174aa2f81a1e7c85b7b7a79eeb044d44dd1d3d76203226b747bb80947b",
        "commitTimestamp": 1598760152,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x4bd6d159fa7e8148a1bf1791cc15d9ceb44f32aec0a3b093f6d09e13184d9962",
        "ethToPay": 0.030873764151998086,
        "umaToPay": 1.6728424852577293
    },
    "0xDb2808ed6cF0B8B9686222689CF4f6e88E42B80A-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598853897,
        "revealGasUsed": 73489,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x6d914ec237965af2de2d5d3d1d0bcf03b4e9872ce2c1b5cf7d7897cf589057cc",
        "commitTimestamp": 1598774811,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xc83af50e34a873bba0ff5d5bd7e88dade4bb79d32f0c10d729df158b9046b3de",
        "ethToPay": 0.03133449735137689,
        "umaToPay": 1.697806531316224
    },
    "0xF288464959A5dD1671bCE3458a3A4eD0Cc314E3C-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598854170,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x2492d13fea9e93ba5f66e91cc13e41a29f765833bd1bbc9e27cf949ba7c5a48e",
        "commitTimestamp": 1598796229,
        "commitGasUsed": 65960,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x67ddffdf8c97ee3501e41c851d42888800f122b71ca4405a34038d1e95ca3b25",
        "ethToPay": 0.03040285979274133,
        "umaToPay": 1.6473273321724264
    },
    "0xEdc63f732D61Df2556fD3aE84F911a84Ac64B4BA-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598854247,
        "revealGasUsed": 69553,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x7e210d5e2dc6b0cfc986b473e7a694b3a13e88a7c10a5c246f5cf6518eed21fb",
        "commitTimestamp": 1598830734,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x546d323d9dfab7aff93d9e1447ebde7f48965e90440e732ca5f3aa956d97b84f",
        "ethToPay": 0.0304076546703602,
        "umaToPay": 1.6475871344742439
    },
    "0xBe3BF9Bb42bF423695B792250BE206C9A58e2dE1-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598855245,
        "revealGasUsed": 71515,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x23ca9040e351016fa05ea4fd0a914945953b72b2979c91b43f4321447ff3763f",
        "commitTimestamp": 1598775736,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x90566cdd82f5e96427f92c780278e798af3cfb4cdce133169f033e0fc9788dda",
        "ethToPay": 0.030870930659708494,
        "umaToPay": 1.6726889572894406
    },
    "0x1B52A70616Ee7ed4Ea070d19FD18a60e467a5276-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598862572,
        "revealGasUsed": 63868,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x0a8cd3d71d09d317100c635eee7bac369d2c184a8701df38fe31f5a56e3ee92f",
        "commitTimestamp": 1598791377,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xf1f4a70c211be5054faa5d5a66492dc15eb7f6c94af4aea3198e69329f58c149",
        "ethToPay": 0.029062744908196506,
        "umaToPay": 1.5747154827407865
    },
    "0x49a8a045EaE2C9D4a7C7e2919bA957064Bd59252-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598863971,
        "revealGasUsed": 83531,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xb4f6a8aaa55bf265e0693bd6fd97b360e4fe60442df7df6b586a13b3653bdcc3",
        "commitTimestamp": 1598817125,
        "commitGasUsed": 76456,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x109f9b5ab7df7eeb61e4a4935cc944888ef6c9c2125ce89c39a8804f89eadb45",
        "ethToPay": 0.03592466619575699,
        "umaToPay": 1.9465170357944563
    },
    "0x2eA5b7c369cb88c171466Bb6dcf3Cbe1C508FBeA-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598864030,
        "revealGasUsed": 83543,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x2d5f3d29fc429818ceb254591ad0514e479671ac5bb66db766a04b4934e98bc4",
        "commitTimestamp": 1598817094,
        "commitGasUsed": 76432,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xc581e2d5d4ad4ab3017e64ba5f3b0f5e24d0b887251e9166890e8ae14235fbc2",
        "ethToPay": 0.035922414108107606,
        "umaToPay": 1.9463950102493408
    },
    "0xb2A5E135892b92d8554daB9AdC5291ee65A9957b-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598872090,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xf1380839375822602994ba11e5265b048dc09aa2c4ab5c6288d2081a1b744aca",
        "commitTimestamp": 1598784859,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x694a0627945bd74af44f4b80523a1fadeda8b8ee6ea8d6c8cbde951517161826",
        "ethToPay": 0.030410488162649792,
        "umaToPay": 1.6477406624425326
    },
    "0xe311958aC79b41BF3a0595470A964B8EFBe0fb36-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598877304,
        "revealGasUsed": 83543,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xbaaaab2fd8638804608b97b91a8fa2b2017e1ef3c07ef1e62367c4321a462c46",
        "commitTimestamp": 1598812840,
        "commitGasUsed": 76432,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xef5e59d303140320b0c9a75441ad8a1188e61b252ab3c4641eeba3c6cdf3ce81",
        "ethToPay": 0.035922414108107606,
        "umaToPay": 1.9463950102493408
    },
    "0x230d31EEC85F4063a405B0F95bdE509C0d0A8b5D-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598877351,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0xa80057b8adb8d9904f9e242807e58c08eec1eb893d847e598155ecc7083443c4",
        "commitTimestamp": 1598825796,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x84de868ec196498ae0f60b8585c3d675b8d6340a470282e8b8d6aab8a478077a",
        "ethToPay": 0.0308712213620286,
        "umaToPay": 1.6727047085010271
    },
    "0xE7A6d81a4B543c1Bf2BD10CEB74A72689Dc2E4b1-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598879399,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x302380c947df12a45687cf4da72976025b8d203f893a9e605ad9649907e3022e",
        "commitTimestamp": 1598784053,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x261fd4a1cc34cfb3f739d6dc90af55fc9f248b28a4c15575c9b5c8b59c4fbf79",
        "ethToPay": 0.030410488162649792,
        "umaToPay": 1.6477406624425326
    },
    "0x28d804Bf2212E220BC2B7B6252993Db8286dF07f-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598884758,
        "revealGasUsed": 74230,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x2d8e89b38c1d97b4f80926bcdddf169023a8a590800c2f5024e372b13162ae0a",
        "commitTimestamp": 1598827257,
        "commitGasUsed": 65972,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x8cd4efd75e595d7654f47644a0e97647ad70d04761977effec387d667d763cb9",
        "ethToPay": 0.03150692271028971,
        "umaToPay": 1.707149106601347
    },
    "0xBEe33aC5520a2245CBF8288e768A5cC26927cddF-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598891903,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x5e6b299dc7b38787bbb3b345a829926b3d8a24efe486936417c2bed8084a7bad",
        "commitTimestamp": 1598807127,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x7687fd80613547c4b49a4459e48ad72b8cab28ffc40736efea3dc76f4383c5b9",
        "ethToPay": 0.0308712213620286,
        "umaToPay": 1.6727047085010271
    },
    "0xCcD9C5465faC5C0Df86fD9B7d524a5949dEFcef6-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598892939,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x5186e84fcc7004ce95f79d33dd1c65ab7e188f8b88f4df7786b9c3151a2e7bf1",
        "commitTimestamp": 1598754189,
        "commitGasUsed": 65972,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xa37fde55b177f820d6e4e025888a304307e46816c225b36f457d37d0463110ac",
        "ethToPay": 0.03086867857205911,
        "umaToPay": 1.6725669317443252
    },
    "0xE6662ec7FC7c84231B29F0f0da6321673d100854-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598895267,
        "revealGasUsed": 69565,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x382b2415a75b8e3ef3e8c03793019752b496647bf2f6e250a7177dd4b4cfab1d",
        "commitTimestamp": 1598813924,
        "commitGasUsed": 65960,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xd747393fd949e82a692154e6c8d5c82c9f9d63ede49881084541f477d9a2498e",
        "ethToPay": 0.03040285979274133,
        "umaToPay": 1.6473273321724264
    },
    "0xde3aF8bb46d556Df3d7C99CC313c4b5713b4a5a5-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598895483,
        "revealGasUsed": 79619,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x31d497a4b8abd576c12abc2423d196852b8110063806b069a543b79014b41032",
        "commitTimestamp": 1598770684,
        "commitGasUsed": 76456,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x3e1305230050a614967535233363a82cf554c0e0e46391438e2c9ad98e828387",
        "ethToPay": 0.03500094770935,
        "umaToPay": 1.8964669181323515
    },
    "0xEb6257954bb490556599E0f4E305051821a33152-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598896185,
        "revealGasUsed": 63127,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x56aabf4a1b7457920d27172871064511c193130a141d8229dc0d4343b9f3f2f5",
        "commitTimestamp": 1598799882,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xef7258a9043731ec455ff912d448b06d0b1f3076f991b827ce56402decdf0e7e",
        "ethToPay": 0.028890319549283686,
        "umaToPay": 1.5653729074556635
    },
    "0x8bd16DE8938A2ad16794ac7E7502896f156E370C-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598897422,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x9267290402d4d89dbcbbfbb964171a96952a2260bcc451553374818802e14b90",
        "commitTimestamp": 1598818239,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x08f7291899d65195cac0d45bc269d66683f62a6c0a48943bd68a1b170d75fcac",
        "ethToPay": 0.0308712213620286,
        "umaToPay": 1.6727047085010271
    },
    "0x2f840532892F91811f99CdAec703F322A779cc66-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598898046,
        "revealGasUsed": 65009,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x21707b33b0009fb15038464bc0531429a27b85e88192bfcd3fd92b8601113816",
        "commitTimestamp": 1598828457,
        "commitGasUsed": 65996,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0xe80957d246ce3b1766233f66c4137547673728145e19fb72a09b951814f58d2b",
        "ethToPay": 0.029334705590034697,
        "umaToPay": 1.5894512104822693
    },
    "0xEF22364cdabD7F39a3B6Fd836f059336c810A8a9-9252-Admin 10-1598733157": {
        "revealTimestamp": 1598911999,
        "revealGasUsed": 71527,
        "revealGasPrice": "236.124357466",
        "revealEthPrice": "434.10",
        "revealTxn": "0x94b38bab6aac95e7072a17664a6a2108231aba266fa1ce16d23214f90c29242f",
        "commitTimestamp": 1598814483,
        "commitGasUsed": 65984,
        "commitGasPrice": "211.899164124",
        "commitEthPrice": "429.04",
        "commitTxn": "0x0765dc6cebc0cb0942115a73793e74fcc3561c3430e37c1b79c9386193e8e5cd",
        "ethToPay": 0.0308712213620286,
        "umaToPay": 1.6727047085010271
    },
    "0x83B19521180d533DbAE360Be77f95D0da3AEf98E-9254-USDETH-1598918400": {
        "revealTimestamp": 1599180788,
        "revealGasUsed": 317119,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa8201bbf989c106e6795ec4286f5c1c4c8203f0d9d9a77b2cfba2221137a5ed0",
        "commitTimestamp": 1599142412,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xdf2275a8094a46db1b7b0379664178ff65c6112cdd470cfbcdbeebaf7c7e0e11",
        "ethToPay": 0.11672743811091454,
        "umaToPay": 5.630248689346102
    },
    "0x83B19521180d533DbAE360Be77f95D0da3AEf98E-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599180788,
        "revealGasUsed": 317119,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa8201bbf989c106e6795ec4286f5c1c4c8203f0d9d9a77b2cfba2221137a5ed0",
        "commitTimestamp": 1599142412,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xdf2275a8094a46db1b7b0379664178ff65c6112cdd470cfbcdbeebaf7c7e0e11",
        "ethToPay": 0.11672743811091454,
        "umaToPay": 5.630248689346102
    },
    "0x797E1f1765a0071454f443114Bf26a53F798c071-9254-USDETH-1598918400": {
        "revealTimestamp": 1599180845,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xf3b963a5bf752283e6d0259f16b576c50098009bd86dfb96ca95c7bf641ec27f",
        "commitTimestamp": 1599135958,
        "commitGasUsed": 122304,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x99441c74e63b5ad31de4f020d519048887a0d3758b9bfd1bbc06b6a57b667d83",
        "ethToPay": 0.07960015038954274,
        "umaToPay": 3.839445546441491
    },
    "0x797E1f1765a0071454f443114Bf26a53F798c071-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599180845,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xf3b963a5bf752283e6d0259f16b576c50098009bd86dfb96ca95c7bf641ec27f",
        "commitTimestamp": 1599135958,
        "commitGasUsed": 122304,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x99441c74e63b5ad31de4f020d519048887a0d3758b9bfd1bbc06b6a57b667d83",
        "ethToPay": 0.07960015038954274,
        "umaToPay": 3.839445546441491
    },
    "0x49a8a045EaE2C9D4a7C7e2919bA957064Bd59252-9254-USDETH-1598918400": {
        "revealTimestamp": 1599180976,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x3a9169bc8ee75988ba09afb6c9191a8ec38d85fc9562db859eadc87e42b65bc2",
        "commitTimestamp": 1599136831,
        "commitGasUsed": 122316,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x6489380a15cc8c0cbccc89b732bfccd465f3bbeff2d6f9ac8a54d9665a814cad",
        "ethToPay": 0.07960484635990446,
        "umaToPay": 3.8396720525775137
    },
    "0x49a8a045EaE2C9D4a7C7e2919bA957064Bd59252-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599180976,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x3a9169bc8ee75988ba09afb6c9191a8ec38d85fc9562db859eadc87e42b65bc2",
        "commitTimestamp": 1599136831,
        "commitGasUsed": 122316,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x6489380a15cc8c0cbccc89b732bfccd465f3bbeff2d6f9ac8a54d9665a814cad",
        "ethToPay": 0.07960484635990446,
        "umaToPay": 3.8396720525775137
    },
    "0x2eA5b7c369cb88c171466Bb6dcf3Cbe1C508FBeA-9254-USDETH-1598918400": {
        "revealTimestamp": 1599181078,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xc1ecf3d8733ae7757c3b898d9c6997caf73b87114c69ed13625e8dcaaaebb6c7",
        "commitTimestamp": 1599136739,
        "commitGasUsed": 122304,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x30d59f7bd5307a2b270efa2e628c8060fa356514da0320356d886628c21f7ca4",
        "ethToPay": 0.07960015038954274,
        "umaToPay": 3.839445546441491
    },
    "0x2eA5b7c369cb88c171466Bb6dcf3Cbe1C508FBeA-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599181078,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xc1ecf3d8733ae7757c3b898d9c6997caf73b87114c69ed13625e8dcaaaebb6c7",
        "commitTimestamp": 1599136739,
        "commitGasUsed": 122304,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x30d59f7bd5307a2b270efa2e628c8060fa356514da0320356d886628c21f7ca4",
        "ethToPay": 0.07960015038954274,
        "umaToPay": 3.839445546441491
    },
    "0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6-9254-USDETH-1598918400": {
        "revealTimestamp": 1599181839,
        "revealGasUsed": 121152,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xf1a609e0eb9c5b2b6b6fe34d6f04294d5415dbb34314770df71033141f2fe60b",
        "commitTimestamp": 1599133510,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xa94fec6689e7118abedfb088698e9cba347c544c2668f1ea4df9edd00cd0bb73",
        "ethToPay": 0.07119671597680098,
        "umaToPay": 3.4341130354736076
    },
    "0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599181839,
        "revealGasUsed": 121152,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xf1a609e0eb9c5b2b6b6fe34d6f04294d5415dbb34314770df71033141f2fe60b",
        "commitTimestamp": 1599133510,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xa94fec6689e7118abedfb088698e9cba347c544c2668f1ea4df9edd00cd0bb73",
        "ethToPay": 0.07119671597680098,
        "umaToPay": 3.4341130354736076
    },
    "0x6888FF4bE3262F3232e1c022De731680a16dDdA7-9254-USDETH-1598918400": {
        "revealTimestamp": 1599181898,
        "revealGasUsed": 125076,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa61bfad88360e98b62df70cecf79fa9f3fd4d521d6e24a4d4f5e8949b7ceeb99",
        "commitTimestamp": 1599148744,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xbda4e758875299c45ed36735deaba07720518905148e03228517e46618d80ab5",
        "ethToPay": 0.07210822505577276,
        "umaToPay": 3.4780789005715076
    },
    "0x6888FF4bE3262F3232e1c022De731680a16dDdA7-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599181898,
        "revealGasUsed": 125076,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa61bfad88360e98b62df70cecf79fa9f3fd4d521d6e24a4d4f5e8949b7ceeb99",
        "commitTimestamp": 1599148744,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xbda4e758875299c45ed36735deaba07720518905148e03228517e46618d80ab5",
        "ethToPay": 0.07210822505577276,
        "umaToPay": 3.4780789005715076
    },
    "0x8d680d96BEFC53D8B90F76d1e8E99e842b3387A2-9254-USDETH-1598918400": {
        "revealTimestamp": 1599182829,
        "revealGasUsed": 132710,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x96c6928babbcca73720f8844728ad78e6a3eef743ca25a63f1995ed6f2b6030f",
        "commitTimestamp": 1599134240,
        "commitGasUsed": 122280,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x792769633aa13feabdb820d6edfd6e8cca011c660b5a81d3f0d2b56e9040d918",
        "ethToPay": 0.0786792493698475,
        "umaToPay": 3.795026669071546
    },
    "0x8d680d96BEFC53D8B90F76d1e8E99e842b3387A2-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599182829,
        "revealGasUsed": 132710,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x96c6928babbcca73720f8844728ad78e6a3eef743ca25a63f1995ed6f2b6030f",
        "commitTimestamp": 1599134240,
        "commitGasUsed": 122280,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x792769633aa13feabdb820d6edfd6e8cca011c660b5a81d3f0d2b56e9040d918",
        "ethToPay": 0.0786792493698475,
        "umaToPay": 3.795026669071546
    },
    "0x49739691fB5f3992b3f2536f309D955558e75933-9254-USDETH-1598918400": {
        "revealTimestamp": 1599183055,
        "revealGasUsed": 125088,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x174dd245b85f2fa47657feaaefcdcc6e24240f65b82c7fe8c7218479ee1d707f",
        "commitTimestamp": 1599148441,
        "commitGasUsed": 109996,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xe9e3638c6a4aeeb66a5ff4b4dcef02560b856a842318c2217c5020d8d786b447",
        "ethToPay": 0.07210162060458745,
        "umaToPay": 3.4777603404863062
    },
    "0x49739691fB5f3992b3f2536f309D955558e75933-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599183055,
        "revealGasUsed": 125088,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x174dd245b85f2fa47657feaaefcdcc6e24240f65b82c7fe8c7218479ee1d707f",
        "commitTimestamp": 1599148441,
        "commitGasUsed": 109996,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xe9e3638c6a4aeeb66a5ff4b4dcef02560b856a842318c2217c5020d8d786b447",
        "ethToPay": 0.07210162060458745,
        "umaToPay": 3.4777603404863062
    },
    "0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96-9254-USDETH-1598918400": {
        "revealTimestamp": 1599183355,
        "revealGasUsed": 125076,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x18d7ee9d08be4be04f09efa85f60d8da96606d21eec99cb8aa7ebecdb5ff1e5c",
        "commitTimestamp": 1599096732,
        "commitGasUsed": 109996,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x32a28c0a2604cd62da7e56c6fd50464b0d33c9737a1da944b0cd70e196f57f39",
        "ethToPay": 0.07209883311504932,
        "umaToPay": 3.4776258882994626
    },
    "0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599183355,
        "revealGasUsed": 125076,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x18d7ee9d08be4be04f09efa85f60d8da96606d21eec99cb8aa7ebecdb5ff1e5c",
        "commitTimestamp": 1599096732,
        "commitGasUsed": 109996,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x32a28c0a2604cd62da7e56c6fd50464b0d33c9737a1da944b0cd70e196f57f39",
        "ethToPay": 0.07209883311504932,
        "umaToPay": 3.4776258882994626
    },
    "0x120219511b9B0254E7D15014dB83B8752289FdA3-9254-USDETH-1598918400": {
        "revealTimestamp": 1599183922,
        "revealGasUsed": 132710,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xbe61b729d1405149c6f67f0a448131fbb629330abba1113fe28b20185d5d7f0a",
        "commitTimestamp": 1599139313,
        "commitGasUsed": 122328,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xdeb364de7b942fe760e514af777e6e08b04c71b274b6c3a491a29858dc61343f",
        "ethToPay": 0.0786980332512944,
        "umaToPay": 3.795932693615636
    },
    "0x120219511b9B0254E7D15014dB83B8752289FdA3-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599183922,
        "revealGasUsed": 132710,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xbe61b729d1405149c6f67f0a448131fbb629330abba1113fe28b20185d5d7f0a",
        "commitTimestamp": 1599139313,
        "commitGasUsed": 122328,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xdeb364de7b942fe760e514af777e6e08b04c71b274b6c3a491a29858dc61343f",
        "ethToPay": 0.0786980332512944,
        "umaToPay": 3.795932693615636
    },
    "0xBEe33aC5520a2245CBF8288e768A5cC26927cddF-9254-USDETH-1598918400": {
        "revealTimestamp": 1599184638,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xdf52e18f0c2022a1a781937e5ab4f709095341671d6227ec6b905d7fd0922ff0",
        "commitTimestamp": 1599146158,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x78f857a7972018068d3f550a7961911b8a6b361ab9ea56a5ff55552396e4b069",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0xBEe33aC5520a2245CBF8288e768A5cC26927cddF-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599184638,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xdf52e18f0c2022a1a781937e5ab4f709095341671d6227ec6b905d7fd0922ff0",
        "commitTimestamp": 1599146158,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x78f857a7972018068d3f550a7961911b8a6b361ab9ea56a5ff55552396e4b069",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0x1B52A70616Ee7ed4Ea070d19FD18a60e467a5276-9254-USDETH-1598918400": {
        "revealTimestamp": 1599187040,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xcb2ac3a38364fce6e13bcf1d3c322614b15b208f4cd4f6c42b1c3449fc5d7eb1",
        "commitTimestamp": 1599142710,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x27b7f014011b6fe0800fabef8c95f580a2335a11209903abe0f72f320c3f502c",
        "ethToPay": 0.07120889540706257,
        "umaToPay": 3.434700499932496
    },
    "0x1B52A70616Ee7ed4Ea070d19FD18a60e467a5276-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599187040,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xcb2ac3a38364fce6e13bcf1d3c322614b15b208f4cd4f6c42b1c3449fc5d7eb1",
        "commitTimestamp": 1599142710,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x27b7f014011b6fe0800fabef8c95f580a2335a11209903abe0f72f320c3f502c",
        "ethToPay": 0.07120889540706257,
        "umaToPay": 3.434700499932496
    },
    "0x67D24f48f1B58C4d4DcDA57Ba2386DcE76353fA2-9254-USDETH-1598918400": {
        "revealTimestamp": 1599190922,
        "revealGasUsed": 132698,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa69224d00a773a0a9dc475bae06874f4fa7f62f6c00b942d201f588f21631652",
        "commitTimestamp": 1599152323,
        "commitGasUsed": 122292,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x45bf7164727fe841193e887800a23117acef1ca0333585270e2411e1fdf79452",
        "ethToPay": 0.07868115785067109,
        "umaToPay": 3.795118723020725
    },
    "0x67D24f48f1B58C4d4DcDA57Ba2386DcE76353fA2-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599190922,
        "revealGasUsed": 132698,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa69224d00a773a0a9dc475bae06874f4fa7f62f6c00b942d201f588f21631652",
        "commitTimestamp": 1599152323,
        "commitGasUsed": 122292,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x45bf7164727fe841193e887800a23117acef1ca0333585270e2411e1fdf79452",
        "ethToPay": 0.07868115785067109,
        "umaToPay": 3.795118723020725
    },
    "0x605ACc13c07CB2dE5261dc64D315857fDE7d5C5c-9254-USDETH-1598918400": {
        "revealTimestamp": 1599191622,
        "revealGasUsed": 134920,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x6c12079e87f840c26ab84eb45d1055d473e13b28e0e2acdf8d745e3a9f49e1df",
        "commitTimestamp": 1599112651,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xa385d44190ba545b971ec5438829da0d4209bce66f582a00b6107d01befc1429",
        "ethToPay": 0.07439489564022694,
        "umaToPay": 3.5883745111789813
    },
    "0x605ACc13c07CB2dE5261dc64D315857fDE7d5C5c-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599191622,
        "revealGasUsed": 134920,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x6c12079e87f840c26ab84eb45d1055d473e13b28e0e2acdf8d745e3a9f49e1df",
        "commitTimestamp": 1599112651,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xa385d44190ba545b971ec5438829da0d4209bce66f582a00b6107d01befc1429",
        "ethToPay": 0.07439489564022694,
        "umaToPay": 3.5883745111789813
    },
    "0xe311958aC79b41BF3a0595470A964B8EFBe0fb36-9254-USDETH-1598918400": {
        "revealTimestamp": 1599194693,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa740223ee96f372d33084d989cdacc8cb8607fb16e47544d99bf3c62479dc629",
        "commitTimestamp": 1599145818,
        "commitGasUsed": 122316,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x9e5bb3e7f1716f6354eade0933805b07ef165c5687e046c9b1f1193bd112ceab",
        "ethToPay": 0.07960484635990446,
        "umaToPay": 3.8396720525775137
    },
    "0xe311958aC79b41BF3a0595470A964B8EFBe0fb36-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599194693,
        "revealGasUsed": 136634,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xa740223ee96f372d33084d989cdacc8cb8607fb16e47544d99bf3c62479dc629",
        "commitTimestamp": 1599145818,
        "commitGasUsed": 122316,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x9e5bb3e7f1716f6354eade0933805b07ef165c5687e046c9b1f1193bd112ceab",
        "ethToPay": 0.07960484635990446,
        "umaToPay": 3.8396720525775137
    },
    "0xD2A78Bb82389D30075144d17E782964918999F7f-9254-USDETH-1598918400": {
        "revealTimestamp": 1599195431,
        "revealGasUsed": 125088,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xf122d215f979702631632f646aef0fdeb821e306ea257bf175c60f7052e3edaf",
        "commitTimestamp": 1599108595,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x7ca5bb270cafa05a51768221aa6cf3581d8bb75b0abd8676fde511f5533d73e6",
        "ethToPay": 0.07212040448603435,
        "umaToPay": 3.478666365030396
    },
    "0xD2A78Bb82389D30075144d17E782964918999F7f-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599195431,
        "revealGasUsed": 125088,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xf122d215f979702631632f646aef0fdeb821e306ea257bf175c60f7052e3edaf",
        "commitTimestamp": 1599108595,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x7ca5bb270cafa05a51768221aa6cf3581d8bb75b0abd8676fde511f5533d73e6",
        "ethToPay": 0.07212040448603435,
        "umaToPay": 3.478666365030396
    },
    "0x4c39185a078B5666C372538231cB793A0928807b-9254-USDETH-1598918400": {
        "revealTimestamp": 1599196512,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x5a3064c31f7b173c526ad8a5b94e17473129ee68d42b0f262a1075e17e3038b6",
        "commitTimestamp": 1599102590,
        "commitGasUsed": 110032,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x3ba8c9e247fd27ec3fcbd525203e59e25909fbe9bc3db2f93a3233d67430ff37",
        "ethToPay": 0.07120419943670084,
        "umaToPay": 3.4344739937964737
    },
    "0x4c39185a078B5666C372538231cB793A0928807b-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599196512,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x5a3064c31f7b173c526ad8a5b94e17473129ee68d42b0f262a1075e17e3038b6",
        "commitTimestamp": 1599102590,
        "commitGasUsed": 110032,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x3ba8c9e247fd27ec3fcbd525203e59e25909fbe9bc3db2f93a3233d67430ff37",
        "ethToPay": 0.07120419943670084,
        "umaToPay": 3.4344739937964737
    },
    "0xFC119030940852a33Fbb787A6fd086EFCaBE458F-9254-USDETH-1598918400": {
        "revealTimestamp": 1599202204,
        "revealGasUsed": 128786,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x7c9170d171a69277d82f063d900dda307960f3a5d992ad0f3163757a770badab",
        "commitTimestamp": 1599104687,
        "commitGasUsed": 122316,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xb7c0d904e457318258e325b0cba15a932eb418c174bf9bef981e7e5b3d0775ed",
        "ethToPay": 0.0777818282019609,
        "umaToPay": 3.7517403223817136
    },
    "0xFC119030940852a33Fbb787A6fd086EFCaBE458F-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599202204,
        "revealGasUsed": 128786,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x7c9170d171a69277d82f063d900dda307960f3a5d992ad0f3163757a770badab",
        "commitTimestamp": 1599104687,
        "commitGasUsed": 122316,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xb7c0d904e457318258e325b0cba15a932eb418c174bf9bef981e7e5b3d0775ed",
        "ethToPay": 0.0777818282019609,
        "umaToPay": 3.7517403223817136
    },
    "0x7553fa99AB1f429551eeC660708C08E14E30584F-9254-USDETH-1598918400": {
        "revealTimestamp": 1599204762,
        "revealGasUsed": 117240,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x853df30873d1790e64175b2d5303d07418eba576551305f050a841bcca157fd0",
        "commitTimestamp": 1599113934,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x72e371ac66a607a0e54743e3d894fd87408abe84d0ea82b4af3de80d87f9ef6f",
        "ethToPay": 0.0702973863280908,
        "umaToPay": 3.390734634834596
    },
    "0x7553fa99AB1f429551eeC660708C08E14E30584F-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599204762,
        "revealGasUsed": 117240,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x853df30873d1790e64175b2d5303d07418eba576551305f050a841bcca157fd0",
        "commitTimestamp": 1599113934,
        "commitGasUsed": 110044,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x72e371ac66a607a0e54743e3d894fd87408abe84d0ea82b4af3de80d87f9ef6f",
        "ethToPay": 0.0702973863280908,
        "umaToPay": 3.390734634834596
    },
    "0xDb2808ed6cF0B8B9686222689CF4f6e88E42B80A-9254-USDETH-1598918400": {
        "revealTimestamp": 1599206722,
        "revealGasUsed": 125088,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xfc7e480e8622c90f0a52274b1f1ff0bf46734f0c57cc46450ef368950c219f6d",
        "commitTimestamp": 1599119991,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x3059cf1281f5554a24cc57b5b45efe635c21ea44bd3e87b145ee310e668df5c8",
        "ethToPay": 0.0721110125453109,
        "umaToPay": 3.478213352758351
    },
    "0xDb2808ed6cF0B8B9686222689CF4f6e88E42B80A-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599206722,
        "revealGasUsed": 125088,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xfc7e480e8622c90f0a52274b1f1ff0bf46734f0c57cc46450ef368950c219f6d",
        "commitTimestamp": 1599119991,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x3059cf1281f5554a24cc57b5b45efe635c21ea44bd3e87b145ee310e668df5c8",
        "ethToPay": 0.0721110125453109,
        "umaToPay": 3.478213352758351
    },
    "0x099686478448CE02F8E61C6b01904d0548daeAe5-9254-USDETH-1598918400": {
        "revealTimestamp": 1599215365,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xe05ba996d56c89b990b196e6eb641b831f6be6ccd67857fccc9ca735efca9810",
        "commitTimestamp": 1599117758,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xf0832ee46437b63374c8ec2ae59b31fea557af3189abbfeaf4e5bcba781abed7",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0x099686478448CE02F8E61C6b01904d0548daeAe5-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599215365,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xe05ba996d56c89b990b196e6eb641b831f6be6ccd67857fccc9ca735efca9810",
        "commitTimestamp": 1599117758,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xf0832ee46437b63374c8ec2ae59b31fea557af3189abbfeaf4e5bcba781abed7",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0xf307077Dd0e27A382E93f2e3d13A9C6584582332-9254-USDETH-1598918400": {
        "revealTimestamp": 1599224177,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xcb82c5d9420d619cafdf66debc4b86ba6a63476b5492a449bfc9ad88a0479af6",
        "commitTimestamp": 1599163629,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xfef49e50345c626fa8729c63e55cd30deaaedf92a58a414e3a19f0d5fefe7f58",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0xf307077Dd0e27A382E93f2e3d13A9C6584582332-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599224177,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xcb82c5d9420d619cafdf66debc4b86ba6a63476b5492a449bfc9ad88a0479af6",
        "commitTimestamp": 1599163629,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xfef49e50345c626fa8729c63e55cd30deaaedf92a58a414e3a19f0d5fefe7f58",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0xfEc2Ade141272f5271FEbaDFca5d21Ca30fc04a5-9254-USDETH-1598918400": {
        "revealTimestamp": 1599229959,
        "revealGasUsed": 121152,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xdd3bf38707985fd4e73e48f65a20e5abdd702aa2dc046e00064340523e01badb",
        "commitTimestamp": 1599141779,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xc9388d063c8e4fef10713564248cf9ac3237aa54c99b44ccc1f6d9a0e87014d0",
        "ethToPay": 0.07119671597680098,
        "umaToPay": 3.4341130354736076
    },
    "0xfEc2Ade141272f5271FEbaDFca5d21Ca30fc04a5-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599229959,
        "revealGasUsed": 121152,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xdd3bf38707985fd4e73e48f65a20e5abdd702aa2dc046e00064340523e01badb",
        "commitTimestamp": 1599141779,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xc9388d063c8e4fef10713564248cf9ac3237aa54c99b44ccc1f6d9a0e87014d0",
        "ethToPay": 0.07119671597680098,
        "umaToPay": 3.4341130354736076
    },
    "0xde3aF8bb46d556Df3d7C99CC313c4b5713b4a5a5-9254-USDETH-1598918400": {
        "revealTimestamp": 1599232773,
        "revealGasUsed": 128786,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x8ee7c90cb53a5ae8e15bf09cff3d69c0ee067f4e80c3e09e697c4a79e753b636",
        "commitTimestamp": 1599141329,
        "commitGasUsed": 122304,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x94071a8facaf6b77555329d6b8af3d6008afff1e3f11c9072ec22c77303131cc",
        "ethToPay": 0.07777713223159918,
        "umaToPay": 3.751513816245691
    },
    "0xde3aF8bb46d556Df3d7C99CC313c4b5713b4a5a5-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599232773,
        "revealGasUsed": 128786,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x8ee7c90cb53a5ae8e15bf09cff3d69c0ee067f4e80c3e09e697c4a79e753b636",
        "commitTimestamp": 1599141329,
        "commitGasUsed": 122304,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x94071a8facaf6b77555329d6b8af3d6008afff1e3f11c9072ec22c77303131cc",
        "ethToPay": 0.07777713223159918,
        "umaToPay": 3.751513816245691
    },
    "0xBf87C98F8A9c1F2A88b613813Ec1E974105EB615-9254-USDETH-1598918400": {
        "revealTimestamp": 1599237535,
        "revealGasUsed": 136622,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xb159406d781f402ac2bf0d014a8fc0200074701d1c123cdaca797f77bafaed63",
        "commitTimestamp": 1599146216,
        "commitGasUsed": 122280,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x125818b3e5919c24b2deaaa2394e4bcafe089affe56dea924cb7e8940c4dbfd3",
        "ethToPay": 0.07958797095928115,
        "umaToPay": 3.8388580819826026
    },
    "0xBf87C98F8A9c1F2A88b613813Ec1E974105EB615-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599237535,
        "revealGasUsed": 136622,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xb159406d781f402ac2bf0d014a8fc0200074701d1c123cdaca797f77bafaed63",
        "commitTimestamp": 1599146216,
        "commitGasUsed": 122280,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x125818b3e5919c24b2deaaa2394e4bcafe089affe56dea924cb7e8940c4dbfd3",
        "ethToPay": 0.07958797095928115,
        "umaToPay": 3.8388580819826026
    },
    "0xa08acA4A5D1A6813DcAE969dF892581047CFCC30-9254-USDETH-1598918400": {
        "revealTimestamp": 1599245657,
        "revealGasUsed": 117228,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x7765c9f1c65f76acc905d1f812356a36ba2d71a2a8eb60307d9a52dc39cb6e07",
        "commitTimestamp": 1599158192,
        "commitGasUsed": 109996,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x2379242e1eb42cde45c95b94536ff8e40d07487f5eb2c9bafe0232f7b42202d9",
        "ethToPay": 0.07027581495710575,
        "umaToPay": 3.3896941581036626
    },
    "0xa08acA4A5D1A6813DcAE969dF892581047CFCC30-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599245657,
        "revealGasUsed": 117228,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x7765c9f1c65f76acc905d1f812356a36ba2d71a2a8eb60307d9a52dc39cb6e07",
        "commitTimestamp": 1599158192,
        "commitGasUsed": 109996,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x2379242e1eb42cde45c95b94536ff8e40d07487f5eb2c9bafe0232f7b42202d9",
        "ethToPay": 0.07027581495710575,
        "umaToPay": 3.3896941581036626
    },
    "0xed6bf5EEfFdcFf967184747648F1C3b578a2f430-9254-USDETH-1598918400": {
        "revealTimestamp": 1599250007,
        "revealGasUsed": 117240,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xbe73c6490b3e0d5a9a43d04586049eae2c5acba18d676e8df2dda5d95081a2b2",
        "commitTimestamp": 1599107048,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xe6c4002a961893e8c0409e60f055808e2c1bc7d6655c748ac16b01430e8129fc",
        "ethToPay": 0.07028799438736734,
        "umaToPay": 3.390281622562551
    },
    "0xed6bf5EEfFdcFf967184747648F1C3b578a2f430-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599250007,
        "revealGasUsed": 117240,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xbe73c6490b3e0d5a9a43d04586049eae2c5acba18d676e8df2dda5d95081a2b2",
        "commitTimestamp": 1599107048,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0xe6c4002a961893e8c0409e60f055808e2c1bc7d6655c748ac16b01430e8129fc",
        "ethToPay": 0.07028799438736734,
        "umaToPay": 3.390281622562551
    },
    "0x2f840532892F91811f99CdAec703F322A779cc66-9254-USDETH-1598918400": {
        "revealTimestamp": 1599253497,
        "revealGasUsed": 108116,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x02eaec5b06dcd97f82696620471023991ef29e426c51c81b3bb92044ac63bff5",
        "commitTimestamp": 1599169743,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x6b65f3f674db818ca68e4f4d3425e4c0fadabdf1ae54899c6e94947915737ed5",
        "ethToPay": 0.06816857317520156,
        "umaToPay": 3.288053143165701
    },
    "0x2f840532892F91811f99CdAec703F322A779cc66-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599253497,
        "revealGasUsed": 108116,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x02eaec5b06dcd97f82696620471023991ef29e426c51c81b3bb92044ac63bff5",
        "commitTimestamp": 1599169743,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x6b65f3f674db818ca68e4f4d3425e4c0fadabdf1ae54899c6e94947915737ed5",
        "ethToPay": 0.06816857317520156,
        "umaToPay": 3.288053143165701
    },
    "0x966Cf5cd0624f1EfCf21B0abc231A5CcC802B861-9254-USDETH-1598918400": {
        "revealTimestamp": 1599253640,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x13ba4df9bad3cae2ab1f2e332e3666941608b577b5cb844392f7d386deed879c",
        "commitTimestamp": 1599146027,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x029168d99c6e445ba00339286bc25db6b28c167444630f44b714497d316e879f",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0x966Cf5cd0624f1EfCf21B0abc231A5CcC802B861-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599253640,
        "revealGasUsed": 121164,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0x13ba4df9bad3cae2ab1f2e332e3666941608b577b5cb844392f7d386deed879c",
        "commitTimestamp": 1599146027,
        "commitGasUsed": 110020,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x029168d99c6e445ba00339286bc25db6b28c167444630f44b714497d316e879f",
        "ethToPay": 0.07119950346633912,
        "umaToPay": 3.434247487660451
    },
    "0x8bd16DE8938A2ad16794ac7E7502896f156E370C-9254-USDETH-1598918400": {
        "revealTimestamp": 1599254897,
        "revealGasUsed": 121152,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xb9752fbaa02d7f03c341fb9f6736f2bbdfad6d24b07c01e1f1a403c513ff17cd",
        "commitTimestamp": 1599154362,
        "commitGasUsed": 110008,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x42becc42fe2846d263c5c0666f6a8856da87b2877b61b2cdcf64d3f4ff13f791",
        "ethToPay": 0.07119202000643926,
        "umaToPay": 3.433886529337585
    },
    "0x8bd16DE8938A2ad16794ac7E7502896f156E370C-9254-Admin 11-1599089728": {
        "revealTimestamp": 1599254897,
        "revealGasUsed": 121152,
        "revealGasPrice": "232.290794845",
        "revealEthPrice": "385.91",
        "revealTxn": "0xb9752fbaa02d7f03c341fb9f6736f2bbdfad6d24b07c01e1f1a403c513ff17cd",
        "commitTimestamp": 1599154362,
        "commitGasUsed": 110008,
        "commitGasPrice": "391.330863477",
        "commitEthPrice": "382.46",
        "commitTxn": "0x42becc42fe2846d263c5c0666f6a8856da87b2877b61b2cdcf64d3f4ff13f791",
        "ethToPay": 0.07119202000643926,
        "umaToPay": 3.433886529337585
    }
}
```

Reply:
claim:

```
{
    "0x797E1f1765a0071454f443114Bf26a53F798c071-9252-Admin 10-1598733157": {
        "timestamp": 1598918432,
        "gasUsed": 231127,
        "gasPrice": "445.944863506",
        "ethToPay": 0.10306989846755127,
        "umaToPay": 3.078574286870406,
        "claimTxn": "0xcf608cc649c09b8d0205799ad4c0aa209ba3759742b94dd298afd04ffc07f1c1"
    },
    "0xEb6257954bb490556599E0f4E305051821a33152-9252-Admin 10-1598733157": {
        "timestamp": 1598918480,
        "gasUsed": 60737,
        "gasPrice": "445.944863506",
        "ethToPay": 0.02708535317476392,
        "umaToPay": 0.809007024110761,
        "claimTxn": "0x955192cf0d99a7ee08fa6e84433c9eb77aa1fa89da4b8d4c52d352da11ebb26b"
    },
    "0x2eA5b7c369cb88c171466Bb6dcf3Cbe1C508FBeA-9252-Admin 10-1598733157": {
        "timestamp": 1598918570,
        "gasUsed": 94060,
        "gasPrice": "445.944863506",
        "ethToPay": 0.04194557386137436,
        "umaToPay": 1.2528639986805106,
        "claimTxn": "0xd11711b2de67bc63fd09465cbbe5c43b13ca10e047ff108df65b9049abf48891"
    },
    "0x49a8a045EaE2C9D4a7C7e2919bA957064Bd59252-9252-Admin 10-1598733157": {
        "timestamp": 1598918645,
        "gasUsed": 94060,
        "gasPrice": "445.944863506",
        "ethToPay": 0.04194557386137436,
        "umaToPay": 1.2528639986805106,
        "claimTxn": "0x24f1cfd91ec674f1be58b1c783dd0fa34466046b9ef8b467363b1786002eb2f8"
    },
    "0x996d5138843993Cebc554c20d82310a43802aFF1-9252-Admin 10-1598733157": {
        "timestamp": 1598918793,
        "gasUsed": 143231,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06387312874482788,
        "umaToPay": 1.9078137720073167,
        "claimTxn": "0x5d67a3c1beedb0f52ea59953377af4750b9bc58d3753a074a873303fe650649f"
    },
    "0xfEc2Ade141272f5271FEbaDFca5d21Ca30fc04a5-9252-Admin 10-1598733157": {
        "timestamp": 1598918948,
        "gasUsed": 143231,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06387312874482788,
        "umaToPay": 1.9078137720073167,
        "claimTxn": "0xb0b3646f06447f4c17487566b2b6b7be1dd9e2ff9a4d8c8d421a024643c4d3f5"
    },
    "0x49739691fB5f3992b3f2536f309D955558e75933-9252-Admin 10-1598733157": {
        "timestamp": 1598919274,
        "gasUsed": 94060,
        "gasPrice": "445.944863506",
        "ethToPay": 0.04194557386137436,
        "umaToPay": 1.2528639986805106,
        "claimTxn": "0x8933788d97a74b8b18e143edb8909d80a0e568cf99a271206519b75c2d61a2d3"
    },
    "0xc1228273C77a4939bCc42A0D8f7B0077228e7F3c-9252-Admin 10-1598733157": {
        "timestamp": 1598919704,
        "gasUsed": 143219,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06386777740646582,
        "umaToPay": 1.907653933946673,
        "claimTxn": "0x00e1ff82cbca49a52e61243b80c3a74a7f9c57d3484261e10af790b4e1abc35d"
    },
    "0xed6bf5EEfFdcFf967184747648F1C3b578a2f430-9252-Admin 10-1598733157": {
        "timestamp": 1598921174,
        "gasUsed": 141269,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06299818492262911,
        "umaToPay": 1.8816802490920377,
        "claimTxn": "0xbe8dfd014b1ba863315e77976b5fc63c37a27b83815e95ad662dd5b82d991bfe"
    },
    "0x1B52A70616Ee7ed4Ea070d19FD18a60e467a5276-9252-Admin 10-1598733157": {
        "timestamp": 1598922832,
        "gasUsed": 61478,
        "gasPrice": "445.944863506",
        "ethToPay": 0.027415798318621867,
        "umaToPay": 0.8188770243555223,
        "claimTxn": "0x6f9ed05c4f984b166bc2d694cf029af12e465a2548fcc0e12450274f14b7e55d"
    },
    "0xE7A6d81a4B543c1Bf2BD10CEB74A72689Dc2E4b1-9252-Admin 10-1598733157": {
        "timestamp": 1598922925,
        "gasUsed": 92098,
        "gasPrice": "445.944863506",
        "ethToPay": 0.04107063003917559,
        "umaToPay": 1.2267304757652313,
        "claimTxn": "0x4e6be9463bffa4a5586480b039433e1c8c635ff1a87286c2f8471f74f56b9e50"
    },
    "0xD2A78Bb82389D30075144d17E782964918999F7f-9252-Admin 10-1598733157": {
        "timestamp": 1598924213,
        "gasUsed": 143219,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06386777740646582,
        "umaToPay": 1.907653933946673,
        "claimTxn": "0x83d202d54b0dd796630fccd72da7db3b9d6749fb84c24d8f9a15230437c35d45"
    },
    "0x9ae57ce31BDae73A0d3BF701998Af5494a1cEeec-9252-Admin 10-1598733157": {
        "timestamp": 1598924843,
        "gasUsed": 141269,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06299818492262911,
        "umaToPay": 1.8816802490920377,
        "claimTxn": "0xda18990bef007b99e011cd511197156aa7031770c7d6040488d7156e537ea11d"
    },
    "0x230d31EEC85F4063a405B0F95bdE509C0d0A8b5D-9252-Admin 10-1598733157": {
        "timestamp": 1598927405,
        "gasUsed": 92098,
        "gasPrice": "445.944863506",
        "ethToPay": 0.04107063003917559,
        "umaToPay": 1.2267304757652313,
        "claimTxn": "0xf7f6cbca52822ee8f2e376ddcb5a9799866057bfe23418216722dd393cb823d9"
    },
    "0xDb2808ed6cF0B8B9686222689CF4f6e88E42B80A-9252-Admin 10-1598733157": {
        "timestamp": 1598945125,
        "gasUsed": 145193,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06474807256702665,
        "umaToPay": 1.933947294922596,
        "claimTxn": "0xc3dea1278264bbbd4b78ee4b85363969dd0931a3bd5db9583c374408a65fc1c3"
    },
    "0xBe3BF9Bb42bF423695B792250BE206C9A58e2dE1-9252-Admin 10-1598733157": {
        "timestamp": 1598948068,
        "gasUsed": 143231,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06387312874482788,
        "umaToPay": 1.9078137720073167,
        "claimTxn": "0xfef8f66b4ee349867d81066db3c4c18c249e4adecc69ec02ae7c0984462ee542"
    },
    "0xF288464959A5dD1671bCE3458a3A4eD0Cc314E3C-9252-Admin 10-1598733157": {
        "timestamp": 1598973913,
        "gasUsed": 141269,
        "gasPrice": "445.944863506",
        "ethToPay": 0.06299818492262911,
        "umaToPay": 1.8816802490920377,
        "claimTxn": "0x215eccbf1fcca133bb7716338f5c87e015ac8177c1e83754ebe4f3004c4bf225"
    },
    "0x6888FF4bE3262F3232e1c022De731680a16dDdA7-9252-Admin 10-1598733157": {
        "timestamp": 1599020805,
        "gasUsed": 143231,
        "gasPrice": "480.09630321",
        "ethToPay": 0.0687646736050715,
        "umaToPay": 1.898604293265449,
        "claimTxn": "0x6481a9e0637a037016790d1dadbe393f1c788e133d001ec492434aeba517af86"
    },
    "0x605ACc13c07CB2dE5261dc64D315857fDE7d5C5c-9252-Admin 10-1598733157": {
        "timestamp": 1599025178,
        "gasUsed": 90136,
        "gasPrice": "480.09630321",
        "ethToPay": 0.04327396038613656,
        "umaToPay": 1.1948013808307874,
        "claimTxn": "0x0ed61a959766e74f4defb3d5eae5d76f57e3c9a0513ae17df2c5485405a03a35"
    },
    "0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96-9252-Admin 10-1598733157": {
        "timestamp": 1599039629,
        "gasUsed": 94060,
        "gasPrice": "480.09630321",
        "ethToPay": 0.0451578582799326,
        "umaToPay": 1.2468161209832238,
        "claimTxn": "0x3838f8cd5e82e0e08688e331cb1b17897abf76d7f4bf856161a60397942e8e9d"
    },
    "0xde3aF8bb46d556Df3d7C99CC313c4b5713b4a5a5-9252-Admin 10-1598733157": {
        "timestamp": 1599056523,
        "gasUsed": 141269,
        "gasPrice": "480.09630321",
        "ethToPay": 0.06782272465817349,
        "umaToPay": 1.8725969231892308,
        "claimTxn": "0xace6dee4c3d3353dfc133b18ca9294f8bb0cefe79a5bb56b536307ff933d4375"
    },
    "0xBf87C98F8A9c1F2A88b613813Ec1E974105EB615-9252-Admin 10-1598733157": {
        "timestamp": 1599075734,
        "gasUsed": 145193,
        "gasPrice": "480.09630321",
        "ethToPay": 0.06970662255196954,
        "umaToPay": 1.9246116633416672,
        "claimTxn": "0x7560adc4a9917689416c5bf7c74d658b9c27668da6c58cebcfda4fa8c0fead0b"
    },
    "0x14A29ccfE3C1cc777A6f3a2ead6F07ceC5cC5dc1-9252-Admin 10-1598733157": {
        "timestamp": 1599075757,
        "gasUsed": 145193,
        "gasPrice": "480.09630321",
        "ethToPay": 0.06970662255196954,
        "umaToPay": 1.9246116633416672,
        "claimTxn": "0xf3c7d16a0529fbd9c0d68326b8b77ddfd34787fb58bbe7c0b40f5628f2e71af3"
    },
    "0x67D24f48f1B58C4d4DcDA57Ba2386DcE76353fA2-9252-Admin 10-1598733157": {
        "timestamp": 1599088865,
        "gasUsed": 143231,
        "gasPrice": "480.09630321",
        "ethToPay": 0.0687646736050715,
        "umaToPay": 1.898604293265449,
        "claimTxn": "0xe7c6016fc2931cd9fb47669c775d4f1508d0b00819778295a2cae96e2c68087d"
    },
    "0xb2A5E135892b92d8554daB9AdC5291ee65A9957b-9252-Admin 10-1598733157": {
        "timestamp": 1599090986,
        "gasUsed": 90136,
        "gasPrice": "480.09630321",
        "ethToPay": 0.04327396038613656,
        "umaToPay": 1.1948013808307874,
        "claimTxn": "0xf09d5ea87f8764061387338e9f11969c218a45e3be5e2107053b89ad60c0d5aa"
    },
    "0x4c39185a078B5666C372538231cB793A0928807b-9252-Admin 10-1598733157": {
        "timestamp": 1599102491,
        "gasUsed": 92098,
        "gasPrice": "391.330863477",
        "ethToPay": 0.03604078986450475,
        "umaToPay": 0.8652956994085678,
        "claimTxn": "0x7df1982b7cad8dc7ffc8819afd36e3b81b4ef5014d65c929767770b965f2bc75"
    },
    "0x120219511b9B0254E7D15014dB83B8752289FdA3-9252-Admin 10-1598733157": {
        "timestamp": 1599117273,
        "gasUsed": 143231,
        "gasPrice": "391.330863477",
        "ethToPay": 0.056050710906674184,
        "umaToPay": 1.3457096606005405,
        "claimTxn": "0xcac86a717ba9c3a9ba5561c18eed26bb6855bd870d0a17d561103c232a76bae8"
    },
    "0xCcD9C5465faC5C0Df86fD9B7d524a5949dEFcef6-9252-Admin 10-1598733157": {
        "timestamp": 1599142389,
        "gasUsed": 143231,
        "gasPrice": "391.330863477",
        "ethToPay": 0.056050710906674184,
        "umaToPay": 1.3457096606005405,
        "claimTxn": "0x72c5226d5b87b129d22a9762262e83794a3c393718a12130e9fb21c6d5d85549"
    },
    "0xCcD9C5465faC5C0Df86fD9B7d524a5949dEFcef6-9231-Admin 5-1595094121": {
        "timestamp": 1599143263,
        "gasUsed": 57233,
        "gasPrice": "391.330863477",
        "ethToPay": 0.02239703930937914,
        "umaToPay": 0.5377257786732672,
        "claimTxn": "0x262192f7da9068d30c8446bc6422d6cfc6d371c0bc2fd657b89abaf1b191c259"
    },
    "0xCcD9C5465faC5C0Df86fD9B7d524a5949dEFcef6-9231-Admin 8-1595095182": {
        "timestamp": 1599143263,
        "gasUsed": 57233,
        "gasPrice": "391.330863477",
        "ethToPay": 0.02239703930937914,
        "umaToPay": 0.5377257786732672,
        "claimTxn": "0x262192f7da9068d30c8446bc6422d6cfc6d371c0bc2fd657b89abaf1b191c259"
    },
    "0xFC119030940852a33Fbb787A6fd086EFCaBE458F-9254-USDETH-1598918400": {
        "timestamp": 1599264467,
        "gasUsed": 263410,
        "gasPrice": "276.608264542",
        "ethToPay": 0.07286138296300822,
        "umaToPay": 1.5333364736044492,
        "claimTxn": "0x441f5fa13dae7d00fc4878874aa22db2000f0ae67fed7d3380720db49c5604af"
    },
    "0xFC119030940852a33Fbb787A6fd086EFCaBE458F-9254-Admin 11-1599089728": {
        "timestamp": 1599264467,
        "gasUsed": 263410,
        "gasPrice": "276.608264542",
        "ethToPay": 0.07286138296300822,
        "umaToPay": 1.5333364736044492,
        "claimTxn": "0x441f5fa13dae7d00fc4878874aa22db2000f0ae67fed7d3380720db49c5604af"
    },
    "0xfEc2Ade141272f5271FEbaDFca5d21Ca30fc04a5-9254-Admin 11-1599089728": {
        "timestamp": 1599268985,
        "gasUsed": 143231,
        "gasPrice": "276.608264542",
        "ethToPay": 0.0396188783386152,
        "umaToPay": 0.8337622582697652,
        "claimTxn": "0x3604d518561c56a7d3f37b33ebc8635348c40e0afe22276e04975ca18f7369c5"
    },
    "0xfEc2Ade141272f5271FEbaDFca5d21Ca30fc04a5-9254-USDETH-1598918400": {
        "timestamp": 1599269116,
        "gasUsed": 92062,
        "gasPrice": "276.608264542",
        "ethToPay": 0.025465110050265603,
        "umaToPay": 0.5359022908506618,
        "claimTxn": "0x905c7c0988222bca78e0395ef0ab0f370f99cee3c0689d4a442349dc72d46e86"
    },
    "0x4c39185a078B5666C372538231cB793A0928807b-9254-USDETH-1598918400": {
        "timestamp": 1599269249,
        "gasUsed": 144811,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04005591939659156,
        "umaToPay": 0.8429595994044793,
        "claimTxn": "0x37b025a31262096b1f9f6659e6a93e911e837f45b608f633df8b69eae557cf69"
    },
    "0x4c39185a078B5666C372538231cB793A0928807b-9254-Admin 11-1599089728": {
        "timestamp": 1599269249,
        "gasUsed": 144811,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04005591939659156,
        "umaToPay": 0.8429595994044793,
        "claimTxn": "0x37b025a31262096b1f9f6659e6a93e911e837f45b608f633df8b69eae557cf69"
    },
    "0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96-9254-USDETH-1598918400": {
        "timestamp": 1599270839,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0xbe455bc848119e2c07977bb87b4d1b0a463d2384b497984a0eb12c10fea06888"
    },
    "0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96-9254-Admin 11-1599089728": {
        "timestamp": 1599270839,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0xbe455bc848119e2c07977bb87b4d1b0a463d2384b497984a0eb12c10fea06888"
    },
    "0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6-9252-Admin 10-1598733157": {
        "timestamp": 1599272232,
        "gasUsed": 143976,
        "gasPrice": "276.608264542",
        "ethToPay": 0.03982495149569899,
        "umaToPay": 0.8380989792478425,
        "claimTxn": "0x6a9f4e5d6950bfbc152c1927b27831eb61dee9cd7c8b30594f4bfcbae9e4067c"
    },
    "0x49a8a045EaE2C9D4a7C7e2919bA957064Bd59252-9254-USDETH-1598918400": {
        "timestamp": 1599272604,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0x989f0c3b33039a33e4ee5ecdab1c0d246938f1b641c0e7f4eb5d0cfe34a3d536"
    },
    "0x49a8a045EaE2C9D4a7C7e2919bA957064Bd59252-9254-Admin 11-1599089728": {
        "timestamp": 1599272604,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0x989f0c3b33039a33e4ee5ecdab1c0d246938f1b641c0e7f4eb5d0cfe34a3d536"
    },
    "0x2eA5b7c369cb88c171466Bb6dcf3Cbe1C508FBeA-9254-USDETH-1598918400": {
        "timestamp": 1599272604,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0xd1c6193614292c1caf7147dd1d565c1c8bef5b16c806a755411dccf1106ed9be"
    },
    "0x2eA5b7c369cb88c171466Bb6dcf3Cbe1C508FBeA-9254-Admin 11-1599089728": {
        "timestamp": 1599272604,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0xd1c6193614292c1caf7147dd1d565c1c8bef5b16c806a755411dccf1106ed9be"
    },
    "0x797E1f1765a0071454f443114Bf26a53F798c071-9254-USDETH-1598918400": {
        "timestamp": 1599274659,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0xfd3793aa49aaf405be348ade5c2d9b01b3f50d0527f2e3494c11fcdcb674c577"
    },
    "0x797E1f1765a0071454f443114Bf26a53F798c071-9254-Admin 11-1599089728": {
        "timestamp": 1599274659,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0xfd3793aa49aaf405be348ade5c2d9b01b3f50d0527f2e3494c11fcdcb674c577"
    },
    "0x49739691fB5f3992b3f2536f309D955558e75933-9254-USDETH-1598918400": {
        "timestamp": 1599275392,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0x34a3bae1df18c05609db0b674016f9fe30f4f7d0c5969d7d2caac8089575846d"
    },
    "0x49739691fB5f3992b3f2536f309D955558e75933-9254-Admin 11-1599089728": {
        "timestamp": 1599275392,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0x34a3bae1df18c05609db0b674016f9fe30f4f7d0c5969d7d2caac8089575846d"
    },
    "0x605ACc13c07CB2dE5261dc64D315857fDE7d5C5c-9254-Admin 11-1599089728": {
        "timestamp": 1599276184,
        "gasUsed": 141269,
        "gasPrice": "276.608264542",
        "ethToPay": 0.0390761729235838,
        "umaToPay": 0.8223412561771646,
        "claimTxn": "0x6654189b3def1fa57ef4dee2ee5868bac491cdfb29741268a55e2037b1034957"
    },
    "0xD2A78Bb82389D30075144d17E782964918999F7f-9254-Admin 11-1599089728": {
        "timestamp": 1599279408,
        "gasUsed": 145181,
        "gasPrice": "276.608264542",
        "ethToPay": 0.040158264454472105,
        "umaToPay": 0.8451134071385579,
        "claimTxn": "0x85f4a0b79b39f1be602f0da28f97eaca8a670ddad7fbc53f12857b63faf45dbf"
    },
    "0xD2A78Bb82389D30075144d17E782964918999F7f-9254-USDETH-1598918400": {
        "timestamp": 1599279956,
        "gasUsed": 94012,
        "gasPrice": "276.608264542",
        "ethToPay": 0.026004496166122503,
        "umaToPay": 0.5472534397194544,
        "claimTxn": "0x06916ec6821964f8e857876875dd3596c8d72967d3e95086bc274e41a2eddd58"
    },
    "0x7553fa99AB1f429551eeC660708C08E14E30584F-9254-USDETH-1598918400": {
        "timestamp": 1599280988,
        "gasUsed": 142849,
        "gasPrice": "276.608264542",
        "ethToPay": 0.03951321398156016,
        "umaToPay": 0.8315385973118786,
        "claimTxn": "0x5a89c99ce3e2ed30ea6bc6f5fab6d7c48abd24609b429322e8be8013d16efc0b"
    },
    "0x7553fa99AB1f429551eeC660708C08E14E30584F-9254-Admin 11-1599089728": {
        "timestamp": 1599280988,
        "gasUsed": 142849,
        "gasPrice": "276.608264542",
        "ethToPay": 0.03951321398156016,
        "umaToPay": 0.8315385973118786,
        "claimTxn": "0x5a89c99ce3e2ed30ea6bc6f5fab6d7c48abd24609b429322e8be8013d16efc0b"
    },
    "0x6888FF4bE3262F3232e1c022De731680a16dDdA7-9254-USDETH-1598918400": {
        "timestamp": 1599281874,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0x80bff39bdbc18da7a5e91350c31fbbf0e3d419a5a8ea844c24de0b75169d1783"
    },
    "0x6888FF4bE3262F3232e1c022De731680a16dDdA7-9254-Admin 11-1599089728": {
        "timestamp": 1599281874,
        "gasUsed": 146773,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04059862481162296,
        "umaToPay": 0.85438060149708,
        "claimTxn": "0x80bff39bdbc18da7a5e91350c31fbbf0e3d419a5a8ea844c24de0b75169d1783"
    },
    "0x1B52A70616Ee7ed4Ea070d19FD18a60e467a5276-9254-USDETH-1598918400": {
        "timestamp": 1599287034,
        "gasUsed": 93678,
        "gasPrice": "276.608264542",
        "ethToPay": 0.025912109005765478,
        "umaToPay": 0.5453091916567996,
        "claimTxn": "0x0269cf5e150baf3f378771a2f4937435827ad2b670f7ecbb065c9b6ea6575ff9"
    },
    "0x1B52A70616Ee7ed4Ea070d19FD18a60e467a5276-9254-Admin 11-1599089728": {
        "timestamp": 1599287034,
        "gasUsed": 93678,
        "gasPrice": "276.608264542",
        "ethToPay": 0.025912109005765478,
        "umaToPay": 0.5453091916567996,
        "claimTxn": "0x0269cf5e150baf3f378771a2f4937435827ad2b670f7ecbb065c9b6ea6575ff9"
    },
    "0xDb2808ed6cF0B8B9686222689CF4f6e88E42B80A-9254-USDETH-1598918400": {
        "timestamp": 1599290012,
        "gasUsed": 94024,
        "gasPrice": "276.608264542",
        "ethToPay": 0.02600781546529701,
        "umaToPay": 0.5473232929432623,
        "claimTxn": "0xab4c6c645722f07a8939bf9dd4d1458634d26975e4d0a5676d07f6cc0e5fb3bb"
    },
    "0xDb2808ed6cF0B8B9686222689CF4f6e88E42B80A-9254-Admin 11-1599089728": {
        "timestamp": 1599290214,
        "gasUsed": 94060,
        "gasPrice": "276.608264542",
        "ethToPay": 0.02601777336282052,
        "umaToPay": 0.5475328526146862,
        "claimTxn": "0x650c08bfb436b4bc7c49ae844c461e515d5d00798af07eb9e9c2feb3ab75a269"
    },
    "0x8d680d96BEFC53D8B90F76d1e8E99e842b3387A2-9254-USDETH-1598918400": {
        "timestamp": 1599319750,
        "gasUsed": 144811,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04005591939659156,
        "umaToPay": 0.8429595994044793,
        "claimTxn": "0x131a778dd7a48076b6f62525353ac9739911b69fc107fba2e1c9552527848f1a"
    },
    "0x8d680d96BEFC53D8B90F76d1e8E99e842b3387A2-9254-Admin 11-1599089728": {
        "timestamp": 1599319750,
        "gasUsed": 144811,
        "gasPrice": "276.608264542",
        "ethToPay": 0.04005591939659156,
        "umaToPay": 0.8429595994044793,
        "claimTxn": "0x131a778dd7a48076b6f62525353ac9739911b69fc107fba2e1c9552527848f1a"
    },
    "0xBEe33aC5520a2245CBF8288e768A5cC26927cddF-9252-Admin 10-1598733157": {
        "timestamp": 1599321105,
        "gasUsed": 143235,
        "gasPrice": "276.608264542",
        "ethToPay": 0.03961998477167337,
        "umaToPay": 0.8337855426777012,
        "claimTxn": "0xf3694b92dc5baee9b765c09a6f67afa586419cc8bbb4a73d065996b6e2bc84b9"
    },
    "0xEF22364cdabD7F39a3B6Fd836f059336c810A8a9-9252-Admin 10-1598733157": {
        "timestamp": 1599340792,
        "gasUsed": 143976,
        "gasPrice": "276.608264542",
        "ethToPay": 0.03982495149569899,
        "umaToPay": 0.8380989792478425,
        "claimTxn": "0xdf68973a12616af61f7c03ecb4440243b65f2ccb7bed69136a5774fbe78aa3e4"
    },
    "0x67D24f48f1B58C4d4DcDA57Ba2386DcE76353fA2-9254-USDETH-1598918400": {
        "timestamp": 1599351653,
        "gasUsed": 144811,
        "gasPrice": "129.483046259",
        "ethToPay": 0.018750569411812048,
        "umaToPay": 0.4151373713654734,
        "claimTxn": "0x43aa32f51a0f9222250c9af1d78e94e4f62bd12b832775f7ac5f0e62e7548def"
    },
    "0x67D24f48f1B58C4d4DcDA57Ba2386DcE76353fA2-9254-Admin 11-1599089728": {
        "timestamp": 1599351653,
        "gasUsed": 144811,
        "gasPrice": "129.483046259",
        "ethToPay": 0.018750569411812048,
        "umaToPay": 0.4151373713654734,
        "claimTxn": "0x43aa32f51a0f9222250c9af1d78e94e4f62bd12b832775f7ac5f0e62e7548def"
    },
    "0xBEe33aC5520a2245CBF8288e768A5cC26927cddF-9254-USDETH-1598918400": {
        "timestamp": 1599352601,
        "gasUsed": 95640,
        "gasPrice": "129.483046259",
        "ethToPay": 0.01238375854421076,
        "umaToPay": 0.2741762586916317,
        "claimTxn": "0x3a103e1078445d95f9bb27de4d0e4e3ad90b7cc36766feaa4c13684b3aa0c0e5"
    },
    "0xBEe33aC5520a2245CBF8288e768A5cC26927cddF-9254-Admin 11-1599089728": {
        "timestamp": 1599352601,
        "gasUsed": 95640,
        "gasPrice": "129.483046259",
        "ethToPay": 0.01238375854421076,
        "umaToPay": 0.2741762586916317,
        "claimTxn": "0x3a103e1078445d95f9bb27de4d0e4e3ad90b7cc36766feaa4c13684b3aa0c0e5"
    },
    "0x1757D400578cF8A2F3Da8228e4402FB1bb3fc968-9252-Admin 10-1598733157": {
        "timestamp": 1599355534,
        "gasUsed": 137802,
        "gasPrice": "129.483046259",
        "ethToPay": 0.017843022740582717,
        "umaToPay": 0.39504429945863895,
        "claimTxn": "0x27fb91c49e2c13f24e56b0c9e98ca7931f64f6141f1bc2fcf67c77cfb3f1457d"
    },
    "0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6-9254-USDETH-1598918400": {
        "timestamp": 1599361181,
        "gasUsed": 95640,
        "gasPrice": "129.483046259",
        "ethToPay": 0.01238375854421076,
        "umaToPay": 0.2741762586916317,
        "claimTxn": "0xcb2706e25300768df34229e94b8a88bc5980af4c69cedb59d8d55087c62c391f"
    },
    "0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6-9254-Admin 11-1599089728": {
        "timestamp": 1599361181,
        "gasUsed": 95640,
        "gasPrice": "129.483046259",
        "ethToPay": 0.01238375854421076,
        "umaToPay": 0.2741762586916317,
        "claimTxn": "0xcb2706e25300768df34229e94b8a88bc5980af4c69cedb59d8d55087c62c391f"
    },
    "0x56D14fcb4587B5845fA4823aF790672EBF92580d-9252-Admin 10-1598733157": {
        "timestamp": 1599404378,
        "gasUsed": 143976,
        "gasPrice": "129.483046259",
        "ethToPay": 0.018642451068185784,
        "umaToPay": 0.4127436325950059,
        "claimTxn": "0xcc6f3e204bf3fc04fa296f24f0acdbb1248cae4eed0edaa251d1d9c45778703a"
    },
    "0xf307077Dd0e27A382E93f2e3d13A9C6584582332-9254-USDETH-1598918400": {
        "timestamp": 1599424309,
        "gasUsed": 144811,
        "gasPrice": "129.483046259",
        "ethToPay": 0.018750569411812048,
        "umaToPay": 0.4151373713654734,
        "claimTxn": "0x60af59a2aaabf8ce9754d0976c03930769b0837de3c30c4cbf6a1604d6efe300"
    },
    "0xf307077Dd0e27A382E93f2e3d13A9C6584582332-9254-Admin 11-1599089728": {
        "timestamp": 1599424309,
        "gasUsed": 144811,
        "gasPrice": "129.483046259",
        "ethToPay": 0.018750569411812048,
        "umaToPay": 0.4151373713654734,
        "claimTxn": "0x60af59a2aaabf8ce9754d0976c03930769b0837de3c30c4cbf6a1604d6efe300"
    },
    "0x8d07D225a769b7Af3A923481E1FdF49180e6A265-9252-Admin 10-1598733157": {
        "timestamp": 1599487817,
        "gasUsed": 143235,
        "gasPrice": "112.638419991",
        "ethToPay": 0.016133764087410886,
        "umaToPay": 0.33723921944922014,
        "claimTxn": "0x15a0a408873eee7fcf7b09ea2eccb5718b03c04b8096790f4ff1bafd8af39cdf"
    },
    "0x099686478448CE02F8E61C6b01904d0548daeAe5-9254-USDETH-1598918400": {
        "timestamp": 1599491723,
        "gasUsed": 144811,
        "gasPrice": "112.638419991",
        "ethToPay": 0.0163112822373167,
        "umaToPay": 0.3409498279586764,
        "claimTxn": "0x138db73d9eacc45f9d6a4f19fec832d5a6de361e80b29dfd1cc7444ea3cabd47"
    },
    "0x099686478448CE02F8E61C6b01904d0548daeAe5-9254-Admin 11-1599089728": {
        "timestamp": 1599491723,
        "gasUsed": 144811,
        "gasPrice": "112.638419991",
        "ethToPay": 0.0163112822373167,
        "umaToPay": 0.3409498279586764,
        "claimTxn": "0x138db73d9eacc45f9d6a4f19fec832d5a6de361e80b29dfd1cc7444ea3cabd47"
    }
}
```